batch/batch/front_end/front_end.py

```diff
@@ -156,12 +156,15 @@ async def _handle_ui_error(session, f, *args, **kwargs):
         await f(*args, **kwargs)
     except KeyError as e:
         set_message(session, str(e), 'error')
+        log.info(f'ui error: KeyError {e}')
         return True
     except BatchOperationAlreadyCompletedError as e:
         set_message(session, e.message, e.ui_error_type)
+        log.info(f'ui error: BatchOperationAlreadyCompletedError {e.message}')
         return True
     except BatchUserError as e:
         set_message(session, e.message, e.ui_error_type)
+        log.info(f'ui error: BatchUserError {e.message}')
```

Comment:
We already have a *ton* of log messages (in Konrad's job its 6.5% of all CPU use, only 1.7 is due to non-info logs). What's the developer use-case for these logs?

Reply:
I wanted an easier way to tell what error was seen (or should have been seen) by a user in the logs. This is on the front end for UI calls. I don't think it's too many to add this for the rare cases where it would be helpful. For example, when Ben's cancel operation wasn't working. I wanted to know if he should have been getting a session error message (or an error message at all).